MOBILE_CLIENT/IOS/_HIGH/UNSAFE_KEYCHAIN_ACCESSIBILITY/recommendation.md

```diff
@@ -0,0 +1,46 @@
+**1. Restrict Keychain Item Accessibility:**
```

Comment:
Is accessibility used in Apple documentation? Accessibility is typically used for users with disabilities.

Reply:
Yes.
This is a reference: https://developer.apple.com/documentation/security/keychain_services/keychain_items/restricting_keychain_item_accessibility?language=objc